rules/proxy/advanced_ip_and_port_scanner_upd_check.yml

```diff
@@ -0,0 +1,31 @@
+title: Advanced IP/Port Scanner Update Check
+id: 1a9bb21a-1bb5-42d7-aa05-3219c7c8f47d
+status: experimental
+description: Detect update check performed by Advanced IP Scanner and Advanced Port Scanner
+author: Axel Olsson
+references:
+  - https://www.virustotal.com/gui/file/ffbc47543eebe9d314dd9774a9c1af2d9d982fdc9d58bd9e569a31413f6931cf/relations
+  - https://www.virustotal.com/gui/file/012bd037dad835d2f68f3fa6571420401747aaae111089d7e1e18546fa3da524/relations
+tags:
+  - attack.discovery
+date: 2022/08/14
+logsource:
+    category: proxy
+detection:
+    selection:
+      # Example request: http://www.advanced-port-scanner.com/checkupdate.php?lng=en&ver=2-5-3680&beta=n&type=upd&rmode=p&product=aps 
+      # Example request2: http://www.advanced-ip-scanner.com/checkupdate.php?lng=en&ver=2-5-3499&beta=n&type=upd&rmode=p&product=aips 
+      c-uri|contains: '/checkupdate.php?'
+      c-uri-query|contains|all:
+        - 'lng='
+        - 'ver='
+        - 'beta='
+        - 'type='
+        - 'rmode='
+        - 'product='
+    condition: selection
+fields:
+  - c-uri
```

Comment:
The c-uri field isn't the one relevant for further investigations. It's `c-ip` the client machine from which the request was sent.